omp/matrix/coo_kernels.cpp

```diff
@@ -141,12 +141,9 @@ void convert_row_idxs_to_ptrs(std::shared_ptr<const OmpExecutor> exec,
                               const IndexType *idxs, size_type num_nonzeros,
                               IndexType *ptrs, size_type length)
 {
-    convert_idxs_to_ptrs(idxs, num_nonzeros, ptrs, length);
+    convert_sorted_idxs_to_ptrs(idxs, num_nonzeros, ptrs, length);
```

Comment:
You need to include `format_conversion.hpp` in order to have access to this function.